.github/workflows/android-regression.yml

```diff
@@ -145,6 +145,12 @@ jobs:
           ref: ${{ github.event.inputs.BRANCH_TO_CHECKOUT }}
           path: 'forked-session-appium'
 
+      - name: Install LFS and pull files
+        run: |
+          cd forked-session-appium
+          git lfs install
+          git lfs pull
```

Comment:
you should be able to add "lfs: true" to the checkout action on line 145 instead of those lines
https://github.com/actions/checkout/tree/v4/